jupyter_fsspec/handlers.py

```diff
@@ -190,6 +190,7 @@ async def post(self):
         :rtype: dict
         """
         request_data = json.loads(self.request.body.decode("utf-8"))
+        print(f"FTRANSFER data\n{request_data}")
```

Comment:
Should be logging

Reply:
Stray debugging statement, it's been removed 👍